VinylTests/Arbitrary.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Arbitrary.swift
+//  Vinyl
+//
+//  Created by Robert Widmann on 2/20/16.
+//  Copyright © 2016 Velhotes. All rights reserved.
+//
+
+import SwiftCheck
+
+let lowerStringGen = 
+    Gen<Character>.fromElementsIn("a"..."z")
+    .proliferateNonEmpty
+    .map(String.init)
+
+let urlStringGen : Gen<String> = sequence([ 
+    Gen.pure("http://"),
```

Comment:
We could add `https` here. This will have impact in some tests. 
`Gen<String>.fromElementsOf(["http://", "https://"])` 
Does it make sense?